### ISSUE TITLE
[four-point-oh] Access to `events` module in `plugin_loaded()` seems broken?

### ISSUE THREAD

**Issue by deathaxe** [wbond/package_control#1590]
GitSavvy throws the following traceback directly after startup.

```
plugins loaded
Traceback (most recent call last):
  File "C:\Apps\Sublime Text 40xx\Lib\python33\sublime_plugin.py", line 528, in on_api_ready
    plc()
  File "C:\Apps\Sublime Text 40xx\Data\Installed Packages\GitSavvy.sublime-package\git_savvy.py", line 22, in plugin_loaded
    package_control.events.install('GitSavvy') or
AttributeError: 'module' object has no attribute 'events'
```

Package Control is running in python 3.8 plugin host on ST4114 on Windows 10.

I can successfully import `package_control` and access `package_control.events.install('GitSavvy')` via console (py3.3 and py3.8) afterwards, so I guess it's some kind of timing issue?

**Comment by deathaxe** [comment]
GitSavvy just imports `package_control` and expects access to `package_control.events` then.

see: https://github.com/timbrel/GitSavvy/issues/1576

**Comment by FichteFoll** [comment]
Importing `package_control.events` is how it should be done in all situations and the fact that it worked before was an implementation detail (or a bug), similar how you can also `import os` and then use `os.path` but the fact this works is an implementation detail and even mentioned in the docs.

If the PC docs are unclear or misleading about this fact, they should definitely be fixed. If we find out that too many packages use this bad pattern we may consider implementing an auto-import from `package_control/__init__.py` to not break potentially unmaintained packages.

**Comment by deathaxe** [comment]
related with #1642

fixed by 03cf6277eb4e4f6f9657ccf3354ebbc85c9f0a6b
